web/yo/app/scripts/controllers/gene.js

```diff
@@ -388,8 +392,10 @@ angular.module('oncokbApp')
                     if(checkObsolete(mutation)) {
                         continue;
                     }
+                    setOriginalStatus([mutation.name_review]);
```

Comment:
I'm a little confused there, why do you remove all review status? And why is it called several times throughout the function?